src/request_collector/server.py

```diff
@@ -45,47 +24,6 @@ def error_handler(_context, exc_info):
     sys.exit()
 
 
-def setup_matrix(private_key: str, chain_id: int) -> Tuple[GMatrixClient, Room]:
-    available_servers_url = DEFAULT_MATRIX_KNOWN_SERVERS[Environment.DEVELOPMENT]
-    available_servers = get_matrix_servers(available_servers_url)
-
-    def _http_retry_delay() -> Iterable[float]:
-        # below constants are defined in raiden.app.App.DEFAULT_CONFIG
-        return udp_utils.timeout_exponential_backoff(
-            DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF,
-            DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL / 5,
-            DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL,
-        )
-
-    client = make_client(
-        servers=available_servers,
-        http_pool_maxsize=4,
-        http_retry_timeout=40,
-        http_retry_delay=_http_retry_delay,
-    )
-
-    try:
-        login_or_register(client, signer=LocalSigner(private_key=decode_hex(private_key)))
-    except MatrixRequestError:
-        raise ConnectionError('Could not login/register to matrix.')
-    except ValueError:
-        raise ConnectionError('Could not login/register to matrix.')
-
-    try:
-        room_name = make_room_alias(chain_id, MONITORING_BROADCASTING_ROOM)
-        monitoring_room = join_global_room(
-            client=client,
-            name=room_name,
-            servers=available_servers,
-        )
-    except MatrixRequestError:
-        raise ConnectionError('Could not join monitoring broadcasting room.')
-    except TransportError:
-        raise ConnectionError('Could not join monitoring broadcasting room.')
-
-    return client, monitoring_room
-
-
 class RequestCollector(gevent.Greenlet):
```

Comment:
Is this still a real Greenlet? It doesn't do anything itself in it's `run` method.